SU2_CFD/src/output/CFlowIncOutput.cpp

```diff
@@ -473,25 +473,25 @@ void CFlowIncOutput::LoadVolumeData(CConfig *config, CGeometry *geometry, CSolve
   if (nDim == 3)
     SetVolumeOutputValue("COORD-Z", iPoint, Node_Geo->GetCoord(2));
   
-  SetVolumeOutputValue("PRESSURE",    iPoint, Node_Flow->GetSolution(0));
-  SetVolumeOutputValue("VELOCITY-X", iPoint, Node_Flow->GetSolution(1));
-  SetVolumeOutputValue("VELOCITY-Y", iPoint, Node_Flow->GetSolution(2));
+  SetVolumeOutputValue("PRESSURE",   iPoint, Node_Flow->GetSolution(iPoint, 0));
```

Comment:
@talbring, do you think it would make sense for `SetVolumeOutputValue` to operate on an entire column of data? e.g. set the data for Solution(0) on nPoints instead of point by point.

Reply:
If we can make that work without having a loop or something like that in the LoadVolumeData routine, why not. I want to have that routine as simple as possible.